### ISSUE TITLE
Feature wish Yokogawa converter: Improve subset selection

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-tasks-core#636]
Make it easier for Fractal to allow for user selection in the Yokogawa Converter:
- Allow users to specify which wells to process more flexibly
- Allow exclusion instead of inclusion selection (e.g. I want to exclude a well or a channel, instead of selecting which ones to include with the current globs)

Motivation: Have a better way to handle edge-cases with microscopy errors

**Comment by tcompa** [comment]
> Allow users to specify which wells to process more flexibly

Can you provide an example of expected behavior for this feature?
That is: is it about providing a simplified access to selection that can already be done through `image_glob_patterns` , or is it something different?

> Allow exclusion instead of inclusion selection (e.g. I want to exclude a well or a channel, instead of selecting which ones to include with the current globs)

Would this be covered by #348?

**Comment by jluethi** [comment]
Good question if everything is covered technically by exclude glob patterns.

We've noticed that with include-only patterns, we couldn't define everything. Exclude patterns may be the answer.

Recent example cases:
I have wells B05-11, wells C05-11
There is a weird issue in well C07 and I don't want to process it.


Exclusion glob patterns seem like the easiest way to get this done.

----

I've also noticed that glob patterns can be quite a barrier. In the MD converter task, we've started to experiment with having a table behind the scenes that contains columns with wells, field of views etc. and allow users to create queries. We've not had users actually test those though, so I'm not sure yet whether that approach will really make it much easier
